frontend/pages/beta/model/new.tsx

```diff
@@ -94,27 +94,29 @@ export default function NewModel() {
               <Typography component='h2' variant='h6'>
                 Overview
               </Typography>
-              <Stack direction='row' spacing={2}>
+              <Box sx={{ width: '100%' }}>
                 <TeamAndModelSelector
                   setTeamValue={setTeamName}
                   teamValue={teamName}
                   setModelValue={setModelName}
                   modelValue={modelName}
                 />
-              </Stack>
+              </Box>
               <Stack>
                 <FormControl>
-                  <Typography component='label' sx={{ fontWeight: 'bold' }} htmlFor={'new-model-description'}>
-                    Description <span style={{ color: theme.palette.primary.main }}>*</span>
-                  </Typography>
-                  <TextField
-                    id='new-model-description'
-                    required
-                    size='small'
-                    value={description}
-                    onChange={(e) => setDescription(e.target.value)}
-                    data-test='modelDescription'
-                  />
+                  <Box id='new-model-description'>
+                    <RichTextEditor
+                      dataValue={description}
+                      onDataValueChange={(value) => setDescription(value)}
+                      data-test='modelDescription'
+                      ariaLabel='modelDescription'
```

Comment:
This shouldn't be camel case, rename to "Model description"

Reply:
Cypress testing requires camel case for the data-test name as it does not accept spaces as valid characters, so we matched this for consistency with the other uses of data-test. I have removed aria-label now though

Reply:
Sorry to be clear, aria-label should be "Model description", data-test can be whatever you want, if you're not using data-test I wold try and use the id field instead if you can.